Surviver_client/app/src/main/java/com/example/myapplicationtest1/FullscreenActivity.java

```diff
@@ -104,20 +89,44 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
 //                    else {
 //                        helloLayout.setVisibility(View.VISIBLE);
 //                    }
-                    //System.out.println(HttpClient.doGet("http://192.168.254.1:8080/user/findAll"));
+    //System.out.println(HttpClient.doGet("http://192.168.254.1:8080/user/findAll"));
+    private final View.OnTouchListener loginHideTouchListener = new View.OnTouchListener() {
```

Comment:
All the code now have better code styles, variables are much easier to understand now!